### PR TITLE
Give <properties.postgresql> default values for serverName and portNumber

### PR DESCRIPTION
**Pull request by aguibert**

### CODE REVIEW

**Comment by aguibert** [comment]
#build

File: dev/com.ibm.ws.jdbc/src/com/ibm/ws/jdbc/internal/JDBCDriverService.java [hunk 1]
```diff
@@ -1053,6 +1055,28 @@ private void shutdownDerbyEmbedded() {
...
+        Arrays.sort(props, new Comparator<PropertyDescriptor>() {
+            @Override
+            public int compare(PropertyDescriptor p1, PropertyDescriptor p2) {
+                if (p1.getName().equalsIgnoreCase("url")) {
+                    if (p2.getName().equalsIgnoreCase("url"))
+                        return p1.getName().compareTo(p2.getName()); 
+                    else
+                        return 1;
+                }
+                if (p2.getName().equalsIgnoreCase("url"))
```
Comment (njr-11) [inline dev/com.ibm.ws.jdbc/src/com/ibm/ws/jdbc/internal/JDBCDriverService.java]: If we are only doing this for PostgreSQL, do we really need the equalsIgnoreCase?  I imagine its more costly than normal equals.

Comment (aguibert) [reply dev/com.ibm.ws.jdbc/src/com/ibm/ws/jdbc/internal/JDBCDriverService.java]: other JDBC drivers may benefit from this as well, so I kept it generic. I don't think the performance difference will be significant.

**Comment by aguibert** [comment]
Cancelling this PR because the PostgreSQL JDBC driver has built-in defaults for serverName and portNumber, so it is not necessary for us to set default values in our metatype.

**Pull request closed without merging**